### PR TITLE
ST: topic status

### PR DESCRIPTION
**Pull request by Frawless**
### Type of change

- Enhancement / new feature

### Description

New system tests for topic CR status. This PR also address hotfix for `oc` cluster availability check.

### Checklist

- [x] Write tests
- [x] Make sure all tests pass

### CODE REVIEW

**Comment by Frawless** [comment]
@strimzi-ci run tests profile=regression testcase=CustomResourceStatusST test_only

File: systemtest/src/test/java/io/strimzi/systemtest/CustomResourceStatusST.java [hunk 1]
```diff
@@ -307,6 +321,16 @@ void waitForKafkaConnectS2IStatus(String status) {
         LOGGER.info("Kafka ConnectS2I cluster is in desired state: {}", status);
     }
 
+    void waitForKafkaTopic(String status, String topicName) {
```
Comment (Seequick1) [inline systemtest/src/test/java/io/strimzi/systemtest/CustomResourceStatusST.java]: Don't we have these types of methods in `StUtils` or it makes sense to have here ? Why do you decided put it here ?

**Comment by Frawless** [comment]
@strimzi-ci run tests profile=regression testcase=CustomResourceStatusST test_only

File: systemtest/src/test/java/io/strimzi/systemtest/CustomResourceStatusST.java (hunk 1 above)
Comment (Frawless) [reply systemtest/src/test/java/io/strimzi/systemtest/CustomResourceStatusST.java]: After discussion, we will address this as part of future refactor in October.

**Pull request merged**